presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkQueryExecutionFactory.java

```diff
@@ -333,6 +333,8 @@ public IPrestoSparkQueryExecution create(
 
         QueryId queryId = queryIdGenerator.createNextQueryId();
         log.info("Starting execution for presto query: %s", queryId);
+        System.out.println(format("Presto on Spark query id: %s, query: %s...", queryId, sql.substring(0, min(30, sql.length()))));
```

Comment:
I would recommend not to add the query test to the standard output. It is not possible to disable it with logging level configuration, and it is actually already printed to the system err.

How about simply `System.out.printf("Query id: %s\n", queryId);`?

Reply:
It is added to give a hint what the query is and save debugging time.   I think it has value. Is safety a concern to print the beginning of query text?

Reply:
It is better not to flood the stdout, as it will generate a lot of logs during testing (potentially making our testing infra to fail) and also it will be more dificuilt to extract useful infromation from std.out, that currently contains only the most important information